.github/workflows/transform-tokens.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       # this creates a json file from the stringified json from the request body
       - name: create json from request body
         id: create-json
-        uses: jsdaniell/create-json@b8e77fa01397ca39cc4a6198cc29a3be5481afef # v1.2.3
+        uses: jsdaniell/create-json@b8e77fa01397ca39cc4a6198cc29a3be5481afef # v.1.2.3
```

Comment:
nit - this should be reverted - just formats the version incorrectly and doesn't change the revision

Reply:
it is actually correct and incorrect at the same time. Looks like the author pushed a wrong tag

<img width="1624" alt="Screenshot 2025-03-27 at 19 11 48" src="https://github.com/user-attachments/assets/3ef59b61-bd12-455e-9d28-d3f072f69c07" />

Reply:
lol :laughing:

Reply:
well thats terrifying